### PR TITLE
[doc][ybm] Early Access track

### PR DESCRIPTION
**Pull request by ddhodge**
Early Access track
CLOUDGA-21942

@netlify /preview/yugabyte-cloud/cloud-basics/create-clusters-overview/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/release-notes.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ On this page:
 
 ## Change log
 
+### July 10, 2024
+
+**Database**
+
+- Ability to choose the latest [stable release](../../faq/yugabytedb-managed-faq/#what-version-of-yugabytedb-does-my-cluster-run-on) when creating a dedicated or Sandbox cluster using the new Early Access track. Early Access has more frequent updates for faster access to the most recent stable version of YugabyteDB. Currently features YugabyteDB v2024.1.0.0.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/release-notes.md:26]: Maybe link to /preview/releases/ybdb-releases/v2024.1/

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-cluster.md [hunk 2]
```diff
@@ -164,7 +164,7 @@ Update the specified cluster.
 | --cloud-provider | Cloud provider. `AWS`, `AZURE`, or `GCP`. |
 | --cluster-tier | Type of cluster. `Sandbox` or `Dedicated`. |
 | --cluster-type | Deployment type. `SYNCHRONOUS` or `GEO_PARTITIONED`. |
-| &#8209;&#8209;database&#8209;version | Database version to use for the cluster. `Innovation`, `Production`, or `Preview`. |
+| &#8209;&#8209;database&#8209;version | Database version to use for the cluster. `Innovation`, `Production`, `Preview`, or `'Early Access'`. |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-cluster.md:167]:
```suggestion
| &#8209;&#8209;database&#8209;version | Database version to use for the cluster. `Innovation`, `Production`, `Preview`, or `Early Access`. |
```

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-cluster.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ Create a cluster.
 | --cluster-tier | Type of cluster. `Sandbox` or `Dedicated`. |
 | --cluster-type | Deployment type. `SYNCHRONOUS` or `GEO_PARTITIONED`. |
 | --credentials | Required. Database credentials for the default user, provided as key-value pairs.<br>Arguments:<ul><li>username</li><li>password</li></ul> |
-| &#8209;&#8209;database&#8209;version | Database version to use for the cluster. `Innovation`, `Production`, or `Preview`. |
+| &#8209;&#8209;database&#8209;version | Database version to use for the cluster. `Innovation`, `Production`, `Preview`, or `'Early Access'`. |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-cluster.md:75]:
```suggestion
| &#8209;&#8209;database&#8209;version | Database version to use for the cluster. `Innovation`, `Production`, `Preview`, or `Early Access`. |
```

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Review by rangasayeelc-yb — APPROVED** [review]

**Pull request merged**